docs/src/Fieldset.doc.js

```diff
@@ -85,6 +85,18 @@ card(
         required: true,
         description: `The content of Fieldset, typically [RadioButtons](/RadioButton), [Checkboxes](/Checkbox) or [TextFields](/TextField).`,
       },
+      {
+        name: 'id',
+        type: 'string',
+        required: false,
```

Comment:
Could you add a description here, maybe "A unique identifier for this Fieldset. `id` must be specified when an errorMessage is added "